pyboy/plugins/game_wrapper_pokemon_gen1.py

```diff
@@ -36,6 +71,22 @@ def post_tick(self):
         self._tile_cache_invalid = True
         self._sprite_cache_invalid = True
 
+        self.number_of_pokemon = self.pyboy.memory[ADDR_NUMBER_OF_POKEMON]
+        self.total_items = self.pyboy.memory[ADDR_TOTAL_ITEMS]
+        # Extract the time in game:
+        _hours = self.pyboy.memory[ADDR_HOURS]
+        _minutes = self.pyboy.memory[ADDR_MINUTES]
```

Comment:
I can see from datacrystal which you link to, that hours and minutes are two bytes. You'll need to check how this works.